.changeset/giant-maps-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-pagerduty': patch
```

Comment:
I dont think I intended for this to be breaking. I am trying to just add the homepage component. Perhaps I need to address the interface change rather than the changeset level?

Reply:
The interface change yeah. Maybe it's possible to make an exclusively additive change, adding the new HomePageX component, and also adding a copy of the old entity based one with the right naming. Then you can deprecate the old one but leave it in place. Same for the types.

Reply:
I tried to make it additive only so I can leave this as patch. Please let me know if it looks ok.